shells/android/java/arcs/storage/BUILD

```diff
@@ -1,3 +1,21 @@
 licenses(["notice"])
 
 package(default_visibility = ["//visibility:public"])
+
+load("//tools/build_defs/kotlin:rules.bzl", "kt_android_library")
```

Comment:
Can you put this at the top of the file?

External buildifier doesn't enforce this yet, unfortunately..